src/components/pages/InstructorHome/InstructorProfile.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Card } from 'antd';
+import { Button } from 'antd';
+
+const { Meta } = Card;
+
+function InstructorProfile() {
+  return (
+    <Card
+      hoverable
+      style={{ width: 240 }}
+      cover={
+        <img
+          alt="example"
+          src="https://i.pinimg.com/474x/db/32/23/db32232ee849096679c32d3392a87694.jpg"
+        />
+      }
+    >
+      <Button type="primary" block>
```

Comment:
Is the button going to make the card expand and show the instructors information or is it going to direct us to another page?

Reply:
I am thinking that the button may lead us to a modal with instructor info on it,   but I think the UX will make the decision later.